server.coffee

```diff
@@ -39,14 +39,18 @@ unless originalPublish
             # If document as already present in oldRelatedPublish then we just set
             # relatedPublish's _documents and call changed to send updated fields
             # (Meteor sends only a diff).
-            if oldRelatedPublish?._documents[collectionName]?[stringId]
```

Comment:
Please things backwards compatible.

Reply:
See example here how this is/was done: https://github.com/peerlibrary/meteor-reactive-publish/pull/42

Reply:
Updated in latest commit.